### ISSUE TITLE
[Bug]: "rank _unit" sometimes returns empty string when called from the server

### ISSUE THREAD

**Issue by jaj22** [official-antistasi-community/A3-Antistasi#1285]
*Version:* Unstable

*Mods:* ACE RHS, not sure if important

*Environment*: MP dedi + HC, not sure if important

***Issue:***
Example from RPT:

```
14:36:57 "3851.95: [Antistasi] | INFO | fn_savePlayer | Stripping saved loadout of player 76561198253025108 due to saving while dead or unconcious"
14:36:57 "3851.95: [Antistasi] | INFO | fn_savePlayer | Saved player 76561198253025108:  rank, 85 money, 0 vehicles"
```

This was the only player saved while dead or unconscious and the only player to have empty-string rank in this example, but that may be a red herring. Could also be related to #1284.

**Comment by jaj22** [comment]
Never seen this again, so either it was caused by escaping out of the player load UI (fixed) or more likely it's a one-off Arma freak that we don't need to deal with. Closing.

**Comment by jaj22** [comment]
Was eventually fixed/worked-around in #2488.
